using-ai-extensions/instructions.md

```diff
@@ -0,0 +1,17 @@
+## Getting Started
+
```

Comment:
This workshop currently has no learning objectives or context so I don't know what the purpose or intent is behind this workshop. Please add a few more details so someone picking this up knows how to use this resource.